src/InternalUtils.vala

```diff
@@ -369,5 +369,9 @@ namespace Gala {
                 }
             });
         }
+
+        public static void bell_notify (Meta.Display display) {
+            display.get_sound_player ().play_from_theme ("bell-window-system", _("Bell Event"), null);
+        }
```

Comment:
This doesn't respect the flash screen option from sound settings, does it?

Reply:
Should be fixed now. Not sure whether there's a better way to use the built in bell?